### ISSUE TITLE
Switch off Raspberries LED's

### ISSUE THREAD

**Issue by sohamakl** [LukeSkywalker92/TeleFrame#58]
I wan't to suggest an option in the Installer script where the user can choose to switch off the Raspberry Pi power and status LED. I have tested with a Raspberry Pi 4. The following commands do the trick.

```
# led0 green, led1 red
sudo sh -c 'echo none > /sys/class/leds/led0/trigger'
sudo sh -c 'echo none > /sys/class/leds/led1/trigger' 
```

**Comment by LukeSkywalker92** [comment]
sounds good. will you implement this?

**Comment by gegu** [comment]
Unfortunately this does not work on a pi3 with current Debian buster.

Can you please check whether the following commands also lead to the desired result?

```
sudo sh -c 'echo 0 > /sys/class/leds/led0/brightness'
sudo sh -c 'echo 0 > /sys/class/leds/led1/brightness'
```
I think there should be a way to configure it, because it won't work on other platforms.

**Comment by mistau** [comment]
checked to use the brightness as proposed by gegu, this works on my pi3b+ using Debian Buster as well.
Will provided an updated patch with config options as soon as we settled #66

**Comment by mistau** [comment]
closed with #89
